src/accountsdb/snapshots.zig

```diff
@@ -370,31 +370,31 @@ pub const HashAge = struct {
     hash_index: u64,
     timestamp: u64,
 
-    pub fn random(rand: std.Random) HashAge {
+    pub fn initRandom(random: std.Random) HashAge {
         return .{
-            .fee_calculator = FeeCalculator.random(rand),
-            .hash_index = rand.int(u64),
-            .timestamp = rand.int(u64),
+            .fee_calculator = FeeCalculator.initRandom(random),
+            .hash_index = random.int(u64),
+            .timestamp = random.int(u64),
         };
     }
 };
 
 pub const BlockhashQueueAges = std.AutoArrayHashMapUnmanaged(Hash, HashAge);
 
 pub fn blockhashQueueAgesRandom(
-    rand: std.Random,
+    random: std.Random,
     allocator: std.mem.Allocator,
     max_list_entries: usize,
 ) std.mem.Allocator.Error!BlockhashQueueAges {
     var ages = BlockhashQueueAges.Managed.init(allocator);
     errdefer ages.deinit();
 
-    try sig.rand.fillHashmapWithRng(&ages, rand, rand.uintAtMost(usize, max_list_entries), struct {
-        pub fn randomKey(_rand: std.Random) !Hash {
-            return Hash.random(_rand);
+    try sig.rand.fillHashmapWithRng(&ages, random, random.uintAtMost(usize, max_list_entries), struct {
+        pub fn randomKey(_random: std.Random) !Hash {
```

Comment:
I would be happy calling these `rand`, but I don't think `_random` is a good name. Ditto all other ones.

Reply:
done b4ffcdef27060b8d7bca2cf85e36c714f9db647c